src/lib/widget/_utilities/api.utilities.ts

```diff
@@ -42,7 +44,7 @@ export function widgetApiFactory(componentApi: ReturnType<typeof _componentApi>)
   }
 
   const configuration = (options?: WidgetConfigOptions) => {
-    if (options?.config) {
+    if (options?.forgerock) {
```

Comment:
Changing `config` property to `forgerock` property, so it's not `config.set({ config: {} }):`, which reads weird. Let me know what you think about this.